### PR TITLE
Fix Query with Pykx Object

### PR DESCRIPTION
**Pull request by marcosvm13**

### CODE REVIEW

File: _posts/2023-07-31-all-roads-lead-to-pykx.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ STATION | MAGNITUDE
 The pythonic interface proved quite convenient, and it was actually extended to many functions from the [q reference card](https://code.kx.com/pykx/1.6/api/q/q.html). However, she noticed the absence of equivalent attributes for operators like `cast`, `drop`, and `exec`, among others. So, she needed to explore alternative methods to be able to express arbitrary q expressions. Yet it proved remarkably straightforward! For instance, the previous qSQL query may also be implemented as follows:
 
 ```python
->>> kx.q("select count distinct MAGNITUDE by STATION from weather")
+>>> kx.q("{select count distinct MAGNITUDE by STATION from}", weather)
```
Comment (jserranohidalgo) [inline _posts/2023-07-31-all-roads-lead-to-pykx.md]: Where is the argument?
`"{select ... STATION from x}", weather)`

**Review by jserranohidalgo — APPROVED** [review]
If it works, ok
but it's a bit strange without the argument x

File: _posts/2023-07-31-all-roads-lead-to-pykx.md (hunk 1 above)
Comment (marcosvm13) [reply _posts/2023-07-31-all-roads-lead-to-pykx.md:140]: I think the two options are valid. Even something like:  `"select ... STATION from ", weather)"` might works. But yes, maybe its clearer with x argument

Comment (marcosvm13) [reply _posts/2023-07-31-all-roads-lead-to-pykx.md]: I try it, and the x is needed. I'm sorry. I thouht it was valid but it seems is not. I fix it now

**Review by jserranohidalgo — APPROVED** [review]

**Comment by jserranohidalgo** [comment]
This should be merged into hablapps/blog/main, not in your fork. I don't know if you can change the target branch ...

**Comment by marcosvm13** [comment]
I dont have permissions. Once you think is fine I can merge to main

**Comment by marcosvm13** [comment]
The target is main i think: image

**Pull request merged**